packages/amplify-category-auth/src/__tests__/provider-utils/awscloudformation/auth-stack-builder/auth-cognito-stack-builder.test.ts

```diff
@@ -17,7 +17,7 @@ describe('generateCognitoStackResources', () => {
     expect(cognitoStack.oAuthCustomResource).toBeDefined();
     expect(
       cognitoStack
-        .oAuthCustomResource!.node!.dependencies!.map((dep: any) => dep.target.logicalId)
+        .oAuthCustomResource!.node!.dependencies!.map((dep: any) => dep.logicalId)
```

Comment:
is this L2 construct?

Reply:
Do you mean by ```oauthCustomresource``` , its is a L1 CFN lambda function but the dependencies which are  snapshot tested here below are L2 construct of CustomResource.

Reply:
Mvn this please. I was wondering where the breaking change came from here. It seems to be core CDK abstractions.